bin/internal/update_dart_sdk.ps1

```diff
@@ -29,7 +29,8 @@ $psMajorVersionLocal = $PSVersionTable.PSVersion.Major
 if ($psMajorVersionLocal -lt $psMajorVersionRequired) {
     Write-Host "Flutter requires PowerShell $psMajorVersionRequired.0 or newer."
     Write-Host "See https://flutter.dev/docs/get-started/install/windows for more."
-    return
+    Write-Host "Current version is $psMajorVersionLocal."
+    exit 2
```

Comment:
Might be worthwhile to leave a comment here why this is two (e.g. link it to the batch file)

Reply:
done